<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adjustments</class>
 <widget class="QWidget" name="adjustments">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>848</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>721</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_3">
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>U</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>V</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="PSCompositeSlider" name="color_effects_v" native="true"/>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Colour effects</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="PSCompositeSlider" name="color_effects_u" native="true"/>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="color_effects_none">
      <property name="text">
       <string>None</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>460</y>
     <width>341</width>
     <height>252</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Image Effects</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>AWB Modes</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="awb_mode"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>DRC Strength</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="drc_strength"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Exposure Mode</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="exposure_mode"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Flash Mode</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="flash_mode"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Meter Mode</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="meter_mode"/>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="image_effect"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>731</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Sharpness</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="PSCompositeSlider" name="sharpness" native="true"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Contrast</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="PSCompositeSlider" name="contrast" native="true"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Saturation</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Brightness</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="PSCompositeSlider" name="brightness" native="true"/>
    </item>
    <item row="2" column="1">
     <widget class="PSCompositeSlider" name="saturation" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PSCompositeSlider</class>
   <extends>QWidget</extends>
   <header location="global">psSliders</header>
   <container>1</container>
   <slots>
    <signal>lnValueChanged(int)</signal>
    <slot>changeCameraValue(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>sharpness</sender>
   <signal>lnValueChanged(int)</signal>
   <receiver>adjustments</receiver>
   <slot>changeCameraValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>1025</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>contrast</sender>
   <signal>lnValueChanged(int)</signal>
   <receiver>adjustments</receiver>
   <slot>changeCameraValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>1025</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saturation</sender>
   <signal>lnValueChanged(int)</signal>
   <receiver>adjustments</receiver>
   <slot>changeCameraValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>1025</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>brightness</sender>
   <signal>lnValueChanged(int)</signal>
   <receiver>adjustments</receiver>
   <slot>changeCameraValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>1025</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>color_effects_u</sender>
   <signal>lnValueChanged(int)</signal>
   <receiver>adjustments</receiver>
   <slot>doColorEffect(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>1025</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>color_effects_v</sender>
   <signal>lnValueChanged(int)</signal>
   <receiver>adjustments</receiver>
   <slot>doColorEffect(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>565</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>1025</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>color_effects_none</sender>
   <signal>stateChanged(int)</signal>
   <receiver>adjustments</receiver>
   <slot>doColorEffect(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>1025</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>awb_mode</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>adjustments</receiver>
   <slot>setAwbMode(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>520</y>
    </hint>
    <hint type="destinationlabel">
     <x>933</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>image_effect</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>adjustments</receiver>
   <slot>setImageEffect(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>468</y>
    </hint>
    <hint type="destinationlabel">
     <x>1025</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drc_strength</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>adjustments</receiver>
   <slot>setDrcStrength(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>552</y>
    </hint>
    <hint type="destinationlabel">
     <x>838</x>
     <y>505</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exposure_mode</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>adjustments</receiver>
   <slot>setExposureMode(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>590</y>
    </hint>
    <hint type="destinationlabel">
     <x>764</x>
     <y>563</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>flash_mode</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>adjustments</receiver>
   <slot>setFlashMode(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>620</y>
    </hint>
    <hint type="destinationlabel">
     <x>1025</x>
     <y>550</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>meter_mode</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>adjustments</receiver>
   <slot>setMeterMode(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>668</y>
    </hint>
    <hint type="destinationlabel">
     <x>1025</x>
     <y>636</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>InValueChanged(int)</signal>
  <slot>changeCameraValue(int)</slot>
  <slot>doColorEffect(int)</slot>
  <slot>setAwbMode(QString)</slot>
  <slot>setImageEffect(QString)</slot>
  <slot>setDrcStrength(QString)</slot>
  <slot>setExposureMode(QString)</slot>
  <slot>setFlashMode(QString)</slot>
  <slot>setMeterMode(QString)</slot>
 </slots>
</ui>
